Atomic/mount.py

```diff
@@ -636,6 +643,9 @@ def _clean_temp_container_by_path(self, path):
             self.d.remove_container(short_cid)
         self._clean_tmp_image()
 
+    def get_driver(self):
```

Comment:
I was using it but figured an easier way.  But I thought it was a worth while function to add.